### PR TITLE
New photo uploader workflow

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: app/helpers/person_image_controller_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module PersonImageControllerHelper
```
Comment (jsugarman) [inline app/helpers/person_image_controller_helper.rb]: is this needed?

File: spec/controllers/person_image_controller_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (jsugarman) [inline spec/controllers/person_image_controller_controller_spec.rb]: is this needed? and file name is wrong if it is anyway by the looks of it

File: spec/helpers/person_image_controller_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (jsugarman) [inline spec/helpers/person_image_controller_helper_spec.rb]: needed?

File: app/helpers/person_image_controller_helper.rb (hunk 1 above)
Comment (stephenrichards) [inline app/helpers/person_image_controller_helper.rb]: Deleted

File: spec/controllers/person_image_controller_controller_spec.rb (hunk 2 above)
Comment (stephenrichards) [inline spec/controllers/person_image_controller_controller_spec.rb]: deleted - will add tests for this controller later

File: spec/helpers/person_image_controller_helper_spec.rb (hunk 3 above)
Comment (stephenrichards) [inline spec/helpers/person_image_controller_helper_spec.rb]: deleted

**Comment by jsugarman** [comment]
👍

**Pull request merged**